nnstreamer_example/custom_example_tensorrt/nnstreamer_customfilter_example_tensorrt_reshape.cc

```diff
@@ -0,0 +1,469 @@
+// SPDX-License-Identifier: LGPL-2.1-only
+
+/**
+ * NNStreamer TensorRT Custom Filter Example: Reshape
+ *
+ * Copyright (C) 2020 Dongju Chae <dongju.chae@samsung.com>
+ *
+ * @file  nnstreamer_customfilter_example_tensorrt_reshape.cc
+ * @date  14 Sep 2020
+ * @brief  TensorRT Custom NNStreamer Filter Example: Reshape
+ * @author  Dongju Chae <dongju.chae@samsung.com>
+ * @bug  No known bugs
+ *
+ * Reference code: sampleDynamicReshape.cpp in TensorRT samples
+ *
+ * This example reshapes the inputs to the given dimensions.
+ * The custom property is to be given as, "custom=D1:D2:D3"
+ * E.g., custom=224:224:3
+ */
+
+#include <glib.h>
+#include <string.h>
+#include <tensor_filter_custom.h>
+#include <nnstreamer_plugin_api.h>
+
+#include <iostream>
+#include <memory>
+#include <vector>
+
+#include <NvInfer.h>
+#include <NvInferRuntimeCommon.h>
+#include <cuda_runtime_api.h>
+
+using Severity = nvinfer1::ILogger::Severity;
+
+/** @brief a global object of ILogger */
+class Logger : public nvinfer1::ILogger
+{
+  void log(Severity severity, const char* msg) override
+  {
+    switch (severity) {
+      case Severity::kWARNING:
+        g_warning ("%s", msg);
+        break;
+      case Severity::kINFO:
+        g_message ("%s", msg);
+        break;
+      case Severity::kVERBOSE:
+        g_debug ("%s", msg);
+        break;
+      default:
+        g_error ("%s", msg);
```

Comment:
g_error() is always fatal. please consider g_critical() (or this is ok when error message should stop the application.)

Reply:
Thanks, they are just changed to `g_critical`